### DISCUSSION TITLE
Welcome to MyWhereslyClock Discussions!

### DISCUSSION THREAD

**Discussion by WhereslyClock** [WhereslyClock/MyWhereslyClock#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ConanLloyd** [comment]
Hello there.  My wife has always wanted one of these clocks and I started looking a few years ago.  I like your implementation and already have all the electronics and a 3D printer so I'm going for it.

I hope you don't mind me asking a couple questions on this awesome project.

1. Do you have any pics of the build showing where the washer(s) go?
2. Am I crazy or is there no power going to the ESP8266?
3. Do you have any ideas how I can inject a "mortal peril" location for one of my users when the temp drops below 40 degrees F? (My wife doesn't do cold :-) )

**Comment by WhereslyClock** [comment]
Hi sorry I dont monitor the account its been ages since I built it.

I cant remember where the washer goes,   if you email my account
***@***.***  with your email I can add you to the fusion360 files
and you can see the construction.

Yeh that fritzing is close to what I got , In my build Im using a boost
buck converter I have 12v in from a power into that, it spits out 5v, which
goes to the motors,  then I have 5v that goes into a 3v converter,   the
esp and the arduinos are all 3.3v versions I use, converter to go from 5v
to 3v

In the diagram the 5v comes from usb into a converter that takes 5v to 3v
so again all the microprocessors are 3.3v and motors 5v.  I havent tried
that hence the caveats in the text.


Mortal Peril eh ?  Hmmm.  I was condering switching from outtracks to
homeassistant.   So if Im getting this user 1 say isnt tracking location
but temp ? thats one way,   or  one of your users tracks location, but if
temp drops below n then  override location?

In all cases you need intellegence outside the clock to override the mqtt
messge,  so by switching from owntracks to HomeAssistant,  you can control
what it outputs perhaps in automations or nodered ?

Alternatively,  why not forget using a user,   but have a bit of code in
the esp that makes an api call to a weather service to get current temp,
then output an LED on/off,   a cutout in the picture could have a
voldermort picture that lights up
